packages/api/test/logged-in-test.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+const { describe, it } = require('mocha');
+
+describe('flaky express server', () => {
+  it('rejects unauthorized session', async () => {
+    const resp = await fetch('http://0.0.0.0:3000/protected/api/repos');
```

Comment:
I think we probably don't want to add a `protectewd/` prefix to `/api/repos`, better that we test a new route that's protected with admin settings, a good option might be picking one of the deletion routes that @StuartRucker has added in.